app/controllers/application_controller.rb

```diff
@@ -11,6 +12,24 @@ def cors_preflight
 
   private
 
+  rescue_from "Exception" do |exception|
```

Comment:
@kreek I have similar concern with respect to swallowing everything, but it could just be one of those gotchas where as long as developer knows, they can mitigate accordingly -- and the rendered json in the view will have the stack trace in test and development environments -- we could also reraise the exception for development environment if necessary.

I didn't really think of the "getting lazy" aspect -- the alternative would be to generate the JSON API error manually somehow, and I worry if that would be less convenient than raising the appropriate exception having 1-2 required arguments. At some point it is more likely to become "copy and paste" of existing code in which case, I wouldn't worry too much about this lazy factor. Besides, that's what code review is for right?

Perhaps we can communicate to F/E developers that if errors / exceptions are provided in any format other than JSON API, let us know and we will fix it.